test/relayclient/RelayProvider.test.ts

```diff
@@ -336,6 +329,19 @@ contract('RelayProvider', function (accounts) {
       await misbehavingPaymaster.setReturnInvalidErrorCode(false)
       await misbehavingPaymaster.setRevertPreRelayCall(true)
 
+      const gsnConfig: Partial<GSNConfig> = {
+        paymasterAddress: misbehavingPaymaster.address,
+        relayHubAddress: relayHub.address,
+        logLevel: 'error'
+      }
+      // @ts-ignore
+      Object.keys(TestRecipient.events).forEach(function (topic) {
+        // @ts-ignore
+        relayHub.constructor.network.events[topic] = TestRecipient.events[topic]
+      })
```

Comment:
Why?

Reply:
Moved it up because we need to put the paymaster into the config. 
It's true that in test it is not required, but ONLY in tests.

Reply:
I do not unserstand the relationship between the code in lines 337 to 341 to the sentence you've written here